packages/bpk-component-breadcrumb/README.md

```diff
@@ -37,6 +42,8 @@ export default class App extends Component {
 | ------------------ | -------- | -------- | ------------- |
 | children           | node     | true     | -             |
 | label              | string   | true     | -             |
+| schemaMetaData     | array    | false    | null          |
```

Comment:
Ideally this should align to what's used in the actual prop type, so how about:

`arrayOf({url: string, label: string})`

Reply:
Agreed 👍 . I've changed